purchase_invoice_indicator/models/account_move.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    is_invoice_issuer = fields.Boolean(string="Is Invoice", readonly=True, store=True)
```

Comment:
```suggestion
    is_invoice_issuer = fields.Boolean()
```

We should do the same in the 11.0 module (with migration scripts).